src/App.vue

```diff
@@ -10,7 +10,7 @@
 
     <v-main>
       <!-- Provides the application the proper gutter -->
-      <v-container fluid>
+      <v-container fluid class="pa-0">
```

Comment:
se vi chiedete perchè: 
- il container aggiunge un poco di padding a tutta l'app, ma previene l'uso di immagini a piena larghezza come nell'about page. Averle con le bande bianche a lato fa _brutto_

Reply:
Anche se il container è fluid? Non si dovrebbe altrimenti impostare qualche parametro nell'immagine o mettere l'immagine fuori dal container? 

Creare più container nella pagina e tenerli uno sotto l'altro? Mettere tale immagine in un container separato?

Reply:
Anche perché questo è il container di tutta l'applicazione, quindi secondo me ci deve sicuramente essere un modo per non manomettere questo container e dare comunque una larghezza di tutta la pagina ad un suo contenuto. Hai provato a vedere se non rompe qualche altra pagina?

Reply:
Non mi è sembrato rompere altre pagine, ho provato a cercare di mettere l'immagine fuori, ma non ho trovato il modo siccome, come dicevi, è il container dell'app stessa, quindi il padding che impone previene tutti i sotto elementi da raggiungere il bordo